src/__tests__/screens/SearchResultsScreen.test.tsx

```diff
@@ -78,22 +80,5 @@ describe('Search Results Screen', () => {
         const headerText = screen.getByText('Search results for:');
         expect(headerText).toBeInTheDocument();
         expect(within(headerText).getByText('iron man')).toBeInTheDocument();
-        expect(screen.getByText('Load More')).toBeInTheDocument();
-        expect(screen.getByText('Load More')).toHaveAttribute('disabled');
-    });
-    it('a clickable "Load More" button is displayed when `moreToFetch` is true', async () => {
-        vi.mocked(usePaginatedData).mockReturnValue({
-            data: TRENDING_DATA,
-            setData: () => {},
-            loading: false,
-            moreToFetch: true,
-            refetch: () => {},
-        });
-
-        render(<RouterProvider router={router} />);
-
-        await screen.findByTestId('search-results-screen');
-        expect(screen.getByText('Load More')).toBeInTheDocument();
-        expect(screen.getByText('Load More')).not.toHaveAttribute('disabled');
```

Comment:
Ideally we could test for the loading spinner, but I think I am fine to break that out into a new issue as it might be difficult.

Reply:
I removed the tests for the Load More Button since it was replaced with the LoafingIndicator